paper/forestecology.Rmd

```diff
@@ -344,12 +342,13 @@ focal_vs_comp_scbi <- focal_vs_comp_scbi %>%
 focal_vs_comp_scbi
 ```
 
-We then compare the observed and fitted/predicted growths to compute the root mean squared error (RMSE) of our model fit. 
+We then compare the observed and fitted/predicted growths to compute the root mean squared error (RMSE) of our model fit. DAVE NOTE: I rounded all the RMSEs, in ecology convention to not show WAY more digits than resolution of the measurement. 
```

Comment:
Agreed on the pseudo-precision. I'm going to see if I can set this globally for the whole document, instead of setting them individually